test/system/practical_support_behaviors_test.rb

```diff
@@ -105,13 +105,36 @@ class PracticalSupportBehaviorsTest < ApplicationSystemTestCase
       end
     end
   end
+
+  describe 'hiding practical support' do
+    before do
+      @patient.practical_supports.create support_type: 'lodging',
+                                         source: 'Other funds (see notes)',
+                                         created_by: @user
+    end
+
+    it 'can hide the practical support tab' do
+      @config = Config.find_or_create_by(config_key: 'hide_practical_support', config_value: { options: ["Yes"] })
+      go_to_edit_page
+      assert has_no_text? /Practical Support/i
+    end
+
+    it 'shows the practical support tab by default' do
```

Comment:
This test doesn't give us any extra coverage in the present, but since the main test above is a negative test it's important to test the positive case too. 

Reasoning: 
Without checking the positive case, if we changed the text "Practical Support" to "New Text" and then introduced a regression where the text showed, the first test would fail silently. By checking the positive case, we ensure that a test failure occurs in that first step, allowing us to update assertions.

Reply:
I'm convinced, thumbs up emoji